src/interface/src/app/top-bar/top-bar.component.html

```diff
@@ -8,4 +8,12 @@
   </button>
   <span class="top-bar-title">Planscape</span>
   <span class="example-spacer"></span>
+  <button
+    mat-icon-button
```

Comment:
Just a quick note - we should standardize on a number of spaces for indentations. These lines have 2 spaces, while the ones in signup.component.html has 6 after <input.

Reply:
Do you have a preference? I find the 6 spaces for multi-line tags to be slightly more readable, but I'm fine either way.

Reply:
I've usually seen 2 spaces in codesearch, although for the 6 spaces, is that because it's indented based on the # of characters in "<input"? If it's something else like "<mat-radio-group", would the indent end up being 16 spaces?

Reply:
Hmm, yes it would. I see how that ends up being less readable. Let's stick with 2 spaces then? I'll go through and fix the existing places that use more than 2.